BackEnd/routes.js

```diff
@@ -236,6 +238,54 @@ router.get('/getTargetLocation', (req, res) => {
 	    res.send(obj.location);
 	}
     });
+
+router.post('/createAlliance', (req, res) => {
+	var jsonRequestBody = req.body;
+
+	var allianceFields = {};
+	allianceFields.allies = jsonRequestBody.allies;
+	allianceFields.targets = jsonRequestBody.targets;
+
+	var alliance = new Alliance(allianceFields);
+	var allianceId = alliance._id;
+	var allies = JSON.parse(jsonRequestBody.allies);
+
+	alliance.save((err) => {
+		if (err) {
+			console.log("Failed to create alliance - Error: " + err.message);
+			res.status(400).json({
+				error: "Failed to create alliance - Error: " + err.message,
+			});
+			return;
+		}
+	});
+
+	// Update users with their alliance ID
+	for (int i = 0; i < allies.length; i++) {
```

Comment:
javascript doesn't support variable declarations with built-in types, use `let` instead

Reply:
Oops my bad lol